components/iap-enabler/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM google/cloud-sdk:alpine
+
+RUN apk add --update jq
```

Comment:
We should also move the curl to install kubectl from the shell script to the Dockerfile.